atom/browser/osr/osr_render_widget_host_view.cc

```diff
@@ -133,22 +129,15 @@ class AtomCopyFrameGenerator {
             &AtomCopyFrameGenerator::CopyFromCompositingSurfaceHasResult,
             weak_ptr_factory_.GetWeakPtr(), damage_rect));
 
-    request->set_area(gfx::Rect(view_->GetPhysicalBackingSize()));
     view_->GetRootLayer()->RequestCopyOfOutput(std::move(request));
   }
 
-  void set_frame_rate_threshold_us(int frame_rate_threshold_us) {
-    frame_duration_ =
-        base::TimeDelta::FromMicroseconds(frame_rate_threshold_us);
-  }
-
  private:
   void CopyFromCompositingSurfaceHasResult(
       const gfx::Rect& damage_rect,
       std::unique_ptr<viz::CopyOutputResult> result) {
     if (result->IsEmpty() || result->size().IsEmpty() ||
         !view_->render_widget_host()) {
-      OnCopyFrameCaptureFailure(damage_rect);
```

Comment:
This code path no longer indicates failure?

Reply:
I'm gonna take a look at this again, I think I know why I removed it but I'm not quite sure anymore.